### PR TITLE
Hello World

### PR DESCRIPTION
**Pull request by tavizon**
Initial project setup.

Express server serving "Hello World!" with Ping.

### CODE REVIEW

File: word-api/README.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
+```
+
+No additional configuration is needed for local development.
+
+### Run the Service
+
+From the `word-api` directory, run:
+
+```bash
+node app.js
```
Comment (fluxsauce) [inline word-api/README.md]: Consider `pm2` instructions.

File: word-api/app.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+// Am I alive?
+app.get('/ping', (request, response) => response.send('PONG'));
+
+// TODO: require routes dir
+app.get('/', (request, response) => response.send('Hello World!'));
+
+// Start it up
+module.exports = app.listen(config.application.port, () => {
+	console.log('word-api - listening on port 5959');
```
Comment (fluxsauce) [inline word-api/app.js]: Use the configured port value rather than hard-coding it.

File: word-api/config/properties.json.template [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "application": {
+    "port": 5959
+  },
+  "helmet": {
+    "hidePoweredBy": {
+      "setTo": "turkey leftovers"
```
Comment (fluxsauce) [inline word-api/config/properties.json.template:7]: heheh

File: word-api/package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "url": "https://github.com/tavizon/CodingChallenge/issues"
+  },
+  "homepage": "https://github.com/tavizon/CodingChallenge/word-api/#readme",
+  "dependencies": {
+    "express": "^4.16.2",
+    "helmet": "^3.9.0"
+  },
+  "devDependencies": {
+    "eslint-config-kink": "git+ssh://git@github.com/Kink-Com/eslint-config-kink.git",
+    "eslint-plugin-protractor": "^1.40.0"
```
Comment (fluxsauce) [inline word-api/package.json]: Are you using protractor? If not, remove this and the e2e config.

**Review by fluxsauce — CHANGES_REQUESTED** [review]
Excellent start! A couple small changes.

File: word-api/app.js (hunk 2 above)
Comment (tavizon) [reply word-api/app.js]: Fixed

File: word-api/config/properties.json.template (hunk 3 above)
Comment (tavizon) [reply word-api/config/properties.json.template:7]: Yum

**Comment by tavizon** [comment]
Comments Addressed.

**Review by fluxsauce — APPROVED** [review]
I used https://httpie.org/ to test.

In future PRs, provide testing instructions, please.

```
http :5959
```

screen shot 2017-12-01 at 12 20 34 pm

```
CodingChallenge/word-api [tavizon] » ./node_modules/.bin/eslint .
CodingChallenge/word-api [tavizon] »
```

Work as advertised, Minimal Viable Product success!

**Comment by tavizon** [comment]
Sweet. Adding httpie to my notes. 
Starting a new branch for the next stage.

**Pull request merged**